e2e/optimism-stack.Dockerfile

```diff
@@ -23,24 +23,40 @@ RUN apt-get install -y jq nodejs npm netcat-openbsd
 
 RUN npm install -g pnpm
 
+
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
+ENV PATH="${PATH}:/root/.cargo/bin"
+
+WORKDIR /git
+RUN git clone https://github.com/casey/just
+WORKDIR /git/just
+RUN cargo install just
+
+WORKDIR /git
+
+RUN curl -L https://foundry.paradigm.xyz | bash
+
+RUN . /root/.bashrc
+
+ENV PATH="${PATH}:/root/.foundry/bin"
+
+RUN foundryup
+
+RUN forge --help
+
 WORKDIR /git
 COPY --from=build_1 /git/op-geth /git/op-geth
 WORKDIR /git
 RUN git clone https://github.com/hemilabs/optimism
 WORKDIR /git/optimism
+RUN echo 6666666ddd
 RUN git fetch origin
 RUN git checkout 7bb2a14f63d01bcb4de3ab3165b007fd85a6b1f9
```

Comment:
Could we move the commit hashes to `ARG`s at the top of the file, so they're easy to update and obvious what they are?